### DISCUSSION TITLE
Ubuntu 2204 image creation getting terminated without the image being created.

### DISCUSSION THREAD

**Discussion by acshankar91** [actions/virtual-environments#9646] [General questions]
![image](https://github.com/actions/runner-images/assets/81577357/7d54d968-4829-4889-bbc5-cacc209207f8)

**Comment by acshankar91** [comment]
W have done multiple runs for image creation, each time the process gets terminated with similar kind of messages. Please help in understanding the issue

**Reply by erik-bershel** [reply]
Hey @acshankar91!
I'll be glad to help you. First things first. What steps do you take to build the image? Have you read the [instructions](https://github.com/actions/runner-images/blob/main/docs/create-image-and-azure-resources.md) and are you using the `main` branch for the build?

**Reply by erik-bershel** [reply]
As I see your build failing somewhere [here](https://dev.azure.com/github/runner-images/_build/results?buildId=135244&view=logs&j=747417c8-a650-5aed-afed-d9b52dbd7d24&t=f245de8a-85db-5d11-6f6b-a3b45c2b2a6a&l=18100). So, it's something wrong with the `Cargo-outdated` installation.

**Comment by acshankar91** [comment]
Hi Erik,
I have cloned the latest main branch from the GitHub repo.
I have been following same build process as done earlier.
Last month I was able to create a build image but now each run is failing
with some packages installation every time.

**Reply by erik-bershel** [reply]
Does the assembly crash in the same place or in different places each time? In the first case, we will have something to think about, in the second case, these are ordinary transitive problems - sometimes something breaks during assembly.

**Comment by acshankar91** [comment]
Each time in different places I am getting the error.

**Reply by erik-bershel** [reply]
Then you are either simply fatally unlucky today (we usually encounter transitive problems during builds no more than once every three or four builds), or you have network/agent problems. Unfortunately, in both cases I cannot help you. Perhaps someone from the community has already encountered a similar problem.

**Comment by samus-aran** [comment]
We're locking this discussion because it has not had recent activity and/or other members have asked for more information to assist you but received no response. Thank you for helping us maintain a productive and tidy community for all our members.
